docs/docs/055-platforms/_category_.yml

```diff
@@ -0,0 +1,6 @@
+label: Platforms
```

Comment:
I almost wonder if the compatibility matrix belongs in this category.

Reply:
Mmmm. Interesting! Good idea to follow up.